etc/tendrl/tendrl.conf.sample

```diff
@@ -13,11 +13,7 @@ log_level = DEBUG
 
 [alerting]
 # Path to log file
-log_cfg_path = /etc/tendrl/logging.yaml
+log_cfg_path = /etc/tendrl/alerting_logging.yaml
```

Comment:
As I commented on the other patch, alert configuartion can be in a different file